salt/cloud/clouds/gce.py

```diff
@@ -416,7 +440,7 @@ def __get_metadata(vm_):
     Get configured metadata and add 'salt-cloud-profile'.
     """
     md = config.get_cloud_config_value(
-        "metadata", vm_, __opts__, default="{}", search_global=False
+        "ex_metadata", vm_, __opts__, default="{}", search_global=False
```

Comment:
sorry i missed this one in the original review. doesn't this also need to remain metadata and not `ex_metadata`?